Web/Edubase.Services/Nomenclature/NomenclatureService.cs

```diff
@@ -43,6 +43,7 @@ public static string GetEstablishmentsPluralName(GT groupType, eTextCase textCas
                 case GT.MultiacademyTrust:
                 case GT.SingleacademyTrust:
                 case GT.SchoolSponsor:
+                case GT.SecureSingleAcademyTrust:
```

Comment:
This is a collection of group types - does it need to be updated to also include Secure SAT?

https://github.com/DFE-Digital/get-information-about-schools/blob/secure_16-19_groups/Web/Edubase.Web.UI/Models/EstablishmentDetailViewModel.cs#L28-L39

```csharp
        private static Dictionary<int, string> _groupType2FieldLabelMappings = new Dictionary<int, string>
        {
            [(int) eLookupGroupType.SingleacademyTrust] = "Single-academy trust",
            [(int) eLookupGroupType.MultiacademyTrust] = "Academy trust",
            [(int) eLookupGroupType.SchoolSponsor] = "Academy sponsor",
            [(int) eLookupGroupType.Trust] = "Trust",
            [(int) eLookupGroupType.Federation] = "Federation",


            [(int) eLookupGroupType.UmbrellaTrust] = "Umbrella trust",
            [(int) eLookupGroupType.ChildrensCentresCollaboration] = "Childrens' centres collaboration",
            [(int) eLookupGroupType.ChildrensCentresGroup] = "Childrens' centre group"
        };
```

Reply:
I think this is a new story.  Asking Mark to create it.